pkg/resources/spark/statefulset.go

```diff
@@ -15,19 +15,42 @@ import (
 	"github.com/dominodatalab/distributed-compute-operator/pkg/util"
 )
 
+const frameworkConfigMountPath = "/opt/bitnami/spark/conf/spark-defaults.conf"
+
 // NewStatefulSet generates a Deployment configured to manage Spark cluster nodes.
 // The configuration is based the provided spec and the desired Component workload.
 func NewStatefulSet(sc *dcv1alpha1.SparkCluster, comp Component) (*appsv1.StatefulSet, error) {
 	var replicas int32
 	var nodeAttrs dcv1alpha1.SparkClusterNode
+	var volumes []corev1.Volume
+	var volumeMounts []corev1.VolumeMount
+	var ports []corev1.ContainerPort
 
 	switch comp {
 	case ComponentMaster:
 		replicas = 1
 		nodeAttrs = sc.Spec.Master.SparkClusterNode
+		ports = []corev1.ContainerPort{
+			{
+				Name:          "http",
```

Comment:
I think I'd prefer the prefixing approach so that users can `kubectl describe svc <name>` and see a more meaningful name for the exposed ports.